### DISCUSSION TITLE
when the next version of v3.x is planned to release ?

### DISCUSSION THREAD

**Discussion by Maheswaran** [jquery/jquery#4996] [Q&A] [answered]
Hi,

There was security vulnerability reported in the version v3.6.0, by Synopsys detect tool. Please find the description of issue below, and also mentioned that it was recently fixed. 

"jQuery contains commented references to the hijacked domain blindsignals, within the files src/queue/delay.js and test/data/jquery-1.9.1.js (the former referring to a Web Archive version of the original site). Users without awareness of the domain's status could be exposed to unspecified attacks if they attempt to follow the links to the hijacked site."

fixed in this commit.  https://github.com/jquery/jquery/commit/e24f2dcf3f6bda1a672502e0233c732065cbbe89

So when do you plan to release next version ?

Thanks,
Maheswaran

**Comment by timmywil** [comment, marked as answer]
The only [remaining issues](https://github.com/jquery/jquery/issues?q=is%3Aopen+is%3Aissue+milestone%3A3.6.1) are related to focus. A team member is working on those and should have them all fixed with one PR. Once that's done, we'll do a release. I can't give a firm date because it depends on their availability.

It's worth noting that the test jquery file, along with whole test folder, is not distributed in releases, but I've gone ahead and removed those links anyway [here](https://github.com/jquery/jquery/pull/4997). Thanks for pointing that out.

**Reply by russi7** [reply]
Thanks @timmywil ! Are you planning to release a minor version for fixing this security issue? 3.6.0 is getting flagged as a vulnerable version in most of the security scanning tools.
Would be possible to release 3.6.1 having 3.6.0 code plus this commit to fix the vulnerability issue?
Many thanks.

**Comment by Reynolds2013** [comment]
Is there an update of when you plan to release next version ?

**Comment by Maheswaran-SE** [comment]
Hi @timmywil,

Any update on this issue ?  When are you planning to release new version  ?

Many thanks.

**Reply by timmywil** [reply]
Yes, we finally got the release ready. It should be available this week or next.

**Reply by Maheswaran-SE** [reply]
Thanks a lot !

**Reply by robbr** [reply]
Hi @timmywil,

Any update as to when the new release is coming out?

Thanks!

**Comment by mgol** [comment]
jQuery 3.6.1 is out now: https://blog.jquery.com/2022/08/26/jquery-3-6-1-maintenance-release/
